tests/composite_pk/test_filter.py

```diff
@@ -405,10 +414,18 @@ def test_filter_users_by_comments_subquery(self):
         self.assertSequenceEqual(queryset, (self.user_2,))
 
     def test_cannot_cast_pk(self):
-        msg = "Cast does not support composite primary keys."
+        msg = "Cast expression does not support composite primary keys."
         with self.assertRaisesMessage(ValueError, msg):
             Comment.objects.filter(text__gt=Cast(F("pk"), TextField())).count()
 
+    def test_annotate_case_when(self):
```

Comment:
Did you want to change filter to annotate below to match this method name?

Reply:
I'll rename to `filter` here but I think we should eventually rename this whole test module `test_query` to create a single space for all queryset interaction tests.